analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PublicMethodWithMultidimensionalArray.CSharp12.cs

```diff
@@ -5,14 +5,14 @@ namespace Repro_8083
 
     public class PrimaryConstructors
     {
-        public class C1(int[][] a);          // FN, the ctor is publicly exposed
-        public class C2(int[,] a);           // FN
+        public class C1(int[][] a);          // Noncompliant, the ctor is publicly exposed
+        public class C2(int[,] a);           // Noncompliant
         public class C3(params int[] a);     // Compliant, params of int
-        public class C4(params int[][][] a); // FN, params of int[][]
+        public class C4(params int[][][] a); // Noncompliant, params of int[][]
```

Comment:
I would add also:
```cs
public class C0(int[] a); // Compliant
public class C6(params int [][] a); // Compliant
```
even tho they are covered by method tests, just for completeness